util/tooltool/tooltool.py

```diff
@@ -41,8 +41,8 @@
 import tempfile
 import threading
 import time
-import urllib2
-import urlparse
+import urllib2  # pylint: disable=import-error
+import urlparse  # pylint: disable=import-error
```

Comment:
I need to fix lithium's (and hence overhaul funfuzz's) imports before this can work out.